src/frontend/js/components/AshleyEditor/index.tsx

```diff
@@ -104,7 +143,9 @@ export const AshleyEditor = (props: MyEditorProps) => {
   };
 
   const editorChange = (stateEditor: EditorState) => {
-    props.target.value = JSON.stringify(
+    (document.getElementById(
+      props.target,
+    ) as HTMLInputElement).value = JSON.stringify(
```

Comment:
```suggestion
    const $target = document.getElementById(props.target)
    if ($target) {
        $target.value = JSON.stringify(
```

Reply:
Thank you for your feedback,  can you explain why $ in the variable's name, is this a convention?
Unfortunately, even with this rewriting, I need to cast it to HTMLInputElement to access the value property.
I'm not sure of the relevance of testing the existence of this target with an if, this dom element is necessary for the code to work so if it doesn't exist nothing will work, IMO a straight error is preferable. WDYT?

Reply:
My mistake. I did not take context into account. 
It's my convention to prefix variables which stores a DOM element with a `$` (jQuery inheritance 😄)

I agree with your comment about the relevance to test the existence of target.

```
const { value } = document.getElementById(props.target) as HTMLInputElement;
```